OKR/PACKAGE-MANAGERS.md

```diff
@@ -2,6 +2,224 @@
+  * Pro: Pretty simple to implement.
+  * Con: Unlike mount, the files would need to live in IPFS and outside IPFS.
+  * Con: To make this work without timestamps inside IPFS, we'd need the user to pass some kind of "add if modified since" date. That's pretty poor UX.
+
+### Objective: Engage with the package manager community to demonstrate how IPFS meets their needs.
+
+While simply working towards our goal to integrate IPFS and package managers will help drive improvements to the protocol and core implementations, the end goal is to actually _integrate_ IPFS and package managers. To that end, we can't stop at improving the core implementations; we need to demonstrate those improvements to the community.
+
+#### Key Results
+
+The key results for this objective mirror the top three key results in the previous objective.
+
+##### P1: Update Performance
+
+Communicate update performance improvements to the community.
+
+##### P1: Communicate fetch performance improvements
+
+Communicate fetch performance improvements to the community.
+
+##### P3: Communicate add performance improvements
+
+Communicate add performance improvements to the community.
+
+#### Initiatives
+
+In each of these initiatives, we plan to pair with a user/community member.
+
+##### Demo: OpenSuse + Mount
+
+**Owner:** @djdv
+
+Ship a demo of managing an RPM (YaST) repository directly via a mounted IPFS filesystem.
+
+##### Blog: Docker over IPFS
+
+**Owner:** @dirkmc (and friends)
+
+Ship a blog post about speeding up CI by using go-ipfs to distribute docker images.
+
+(docker _is_ a package manager, or close enough)
+
+##### Demo: Demonstrate performance improvements adding NPM to IPFS
+
+**Owner** @aschmahmann/@achingbrain
+
+Ship a demo of adding NPM to IPFS.
+
+TODO: I'd like to find a better way to demonstrate improved add performance.
+
+### Objective: IPFS ships a release.
+
+All this work is useless unless IPFS can ship a release. This will likely mean re-prioritizing as necessary to:
+
+1. Help test-infra build out the testground test network.
+2. Help test-infra build test cases for go-ipfs.
+3. Help fix the DHT (shipping a go-ipfs 0.5.0 release that _doesn't_ fix the DHT is not an option).
+
+#### Key Results
+
+IPFS has shipped 0.5.0 release.
+
+#### Initiatives
+
+Unfortunately, we're not yet at a place where we can fully plan out _how_ this team can help with this project but we can make a decent start:
+
+Roadmap:
+
+1. Testground is up and running and can test a >10K node DHT.
+2. Proposed DHT fixes are tested against various network scenarios and we are confident they will significantly improve the state of the network.
+3. We cut a release.
+
+##### P0: Fix the DHT
+
+@aschmahmann will likely be pulled into working on DHT fixes during the second half of the quarter (once we can actually start _testing_ DHT fixes).
```

Comment:
@aschmahmann - sound good to you?

Reply:
Works for me, seems pretty important since we need it to get a release out.